src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+document.querySelector('.spider').style.cssText = `
```

Comment:
Consider it will be more readable
```suggestion
const spider = document.querySelector('.spider');

spider.style.cssText = `
```